### ISSUE TITLE
Support link has issues on iOS

### ISSUE THREAD

**Issue by pbakaus** [ampproject/docs#1713]
Looks like the "Support" link is hard/impossible to click on iOS, here's a video (provided to me):

[ScreenRecording_04-02-2019 17-44-10.mp4.zip](https://github.com/ampproject/docs/files/3036415/ScreenRecording_04-02-2019.17-44-10.mp4.zip)

**Comment by pbakaus** [comment]
Actually further investigating, this is an issue with the blog, which has a different code base. Will reroute to the right people..

**Comment by swissspidy** [comment]
Thanks for the patience here. The issue has now been resolved.

The issues was that the header HTML included in the blog was differing from the one on the main website. It included a toggle button for the support menu item (to show/hide sub items) when it now is just a regular link without any sub items. I manually updated the markup to fix this.

It would still be great to have an easier way for us to automatically include the right markup for the header in the WordPress theme so we don't have to manually check for updates every time (or wait until we get notified about it).
